src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -1203,14 +1203,18 @@ private static void WriteSimpleValue(bool indented, string jsonIn, string jsonOu
 
         private static void AssertContents(string expectedValue, ArrayBufferWriter<byte> buffer)
         {
-            Assert.Equal(
-                expectedValue,
-                Encoding.UTF8.GetString(
+            string value = Encoding.UTF8.GetString(
                     buffer.WrittenSpan
 #if netfx
                         .ToArray()
 #endif
-                    ));
+                    );
+
+            // Temporary hack until we can use the same escape algorithm throughout.
+            if (expectedValue.NormalizeToJsonNetFormat() != value.NormalizeToJsonNetFormat())
+            {
+                Assert.Equal(expectedValue.NormalizeToJsonNetFormat(), value.NormalizeToJsonNetFormat());
+            }
```

Comment:
Why if+Assert instead of just Assert?  If the `!=` is doing something different than Assert.Equal then a comment explaining a case warranting this style would be good.

Reply:
I'll clean this up to just use Assert. Thanks.